test/ijavascript-base-notebook-latest/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# as of today this is python-3.7.1
+FROM jupyter/minimal-notebook:latest
+
+USER root
+# prerequisites with apt-get
+# we do install python(2) here because
+# some npm build part named gyp still requires it
+RUN apt-get update && apt-get install -y gcc g++ make python
+
+
+# !!! dirty trick!!!
+# original PATH is
+# /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+# move conda's path **at the end**
+# so that python resolves in /usr/bin/python(2)
+# but node is still found in conda
+ENV PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/conda/bin"
+USER jovyan
+RUN npm install -g ijavascript
```

Comment:
I haven't tested the idea, but I think it's possible to avoid the need for updating and restoring `PATH`, by replacing this line with:
```dockerfile
RUN PATH=/usr/bin/:$PATH npm install -g ijavascript
```

Reply:
well, it might work, or not, depending on whether there are forks involved in this npm install command  
I'm unsure if/when I'll be able to try it out anytime soon though, so it's probably best to keep it as a comment for now  
if you agree I can do that